pkg/daemon/daemon.go

```diff
@@ -55,6 +55,14 @@ type networksMap struct {
 	theMap map[types.UID][]*v1.NetworkSelectionElement
 }
 
+// Exponential backoff ~26 sec + 6 * <api call time>
+// NOTE: k8s client has built in exponential backoff, which ib-kubernetes don't use.
+// In case of increased backoff time might be KUBE_CLIENT_BACKOFF_BASE & KUBE_CLIENT_BACKOFF_DURATION
```

Comment:
Its not clear from the comment what the maximal backoff time would be.
is it the maximal value between the two ? or something else ? as we are performing exponential backoff of an exponential backoff method ?

Reply:
i would suggest a reword:
 ```
In this case, there may be an increased execution time as <api call time> may take longer as retries are performed by k8s client.
k8s client backoff time is determined by KUBE_CLIENT_BACKOFF_BASE, KUBE_CLIENT_BACKOFF_DURATION environment variables [1]......
```